tensorboard/plugins/graph/tf_graph_dashboard/tf-graph-dashboard.html

```diff
@@ -90,6 +90,7 @@ <h3>No graph definition files were found.</h3>
           out-stats="{{_stats}}"
           progress="{{_progress}}"
           out-hierarchy-params="{{_hierarchyParams}}"
+          compatibility-provider="{{_compatibilityProvider}}"
```

Comment:
Please use square brackets unless tf-graph-loader notifies which it does not.
```suggestion
          compatibility-provider="[[_compatibilityProvider]]"
```